contract/src/helpers.rs

```diff
@@ -1,5 +1,14 @@
 use crate::*;
 
+//used to generate a unique prefix in our storage collections (this is to avoid data collisions)
+pub(crate) fn hash_account_id(account_id: &AccountId) -> CryptoHash {
+    //get the default hash
+    let mut hash = CryptoHash::default();
+    //we hash the account ID and return it
+    hash.copy_from_slice(&env::sha256(account_id.as_bytes()));
+    hash
```

Comment:
Just use `env::sha256_array` and you don't need to allocate and copy over
```suggestion
    env::sha256_array(account_id.as_bytes())
```